carbonmark-api/src/graphql/marketplace.gql

```diff
@@ -125,3 +113,38 @@ query getProjectsById($key: String, $vintageStr: BigInt) {
     }
   }
 }
+
+query getCreditListings($projectId: String, $vintageStr: BigInt) {
```

Comment:
This name makes me think there is some model (entity) type called CreditListing. Can we add projectId to `findProjects` and then label it's response/result for this use case?

Something like:

```ts
const {data:creditListings} = gqlSdk.marketplace.findProjects({id:...})
```